### ISSUE TITLE
2.5.2: Pointer Cancellation mentions failure for  2.5.3, editorial issue

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#707]
A failure reference for techniques mentions 2.5.3, think that needs to be 2.5.2

https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html#techniques

> @@ Failure of SC 2.5.3 due to activating a button on initial touch location rather than the final touch location.

**Comment by mbgower** [comment]
Included in https://github.com/w3c/wcag/pull/711

**Comment by ronsman** [comment]
This is replicated in https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=252#pointer-cancellation.

@@ Failure of SC 2.5.3 due to activating a button on initial touch location rather than the final touch location.

should be

Failure of Success Criterion 2.5.2 due to activating a control on the down-event

linked to https://www.w3.org/WAI/WCAG21/Techniques/failures/F101.
